sysutils/pfSense-pkg-nut/Makefile

```diff
@@ -2,7 +2,7 @@
 
 PORTNAME=	pfSense-pkg-nut
 PORTVERSION=	2.8.0
-PORTREVISION=	2
+PORTREVISION=	5
```

Comment:
This seems like way too large a change for just a portrevision bump. It warrants a larger increase to give the user a better idea about the impact of the change. Though it doesn't look like they've settled on a new version upstream yet, seems like `2.9.0` is most likely. There really isn't a compelling reason to tie this version to the NUT version. The NUT version is shown in the package info separately these days.

Reply:
I responded in the wrong box. My bad. Please see below.

Reply:
2.8.1 would be fine.

The main problem with trying to keep them aligned is that it forces you to abide by their versions even if you make substantial changes to the package. Ideally the versions should be separate, but I get wanting to tie them together. In the past before we showed the dependencies in the package list it wasn't obvious what was being included, but that hasn't been the case in several years now. Users are shown quite visibly which underlying version would be installed.

Reply:
Thanks Jim. I've changed the version to 2.8.1.